misc/test-client-rust/src/util.rs

```diff
@@ -0,0 +1,122 @@
+use std::{collections::HashMap, io::Write};
+use reqwest::StatusCode;
+use serde_json::Value;
+
+use crate::CliArgs;
+
+// Type alias for response data
+pub type Response = HashMap<String, Value>;
+
+pub fn verbose_log(cli_args: &CliArgs, message: &str) {
+    if cli_args.verbose {
+        println!("{}", message);
+    }
+}
+
+pub fn make_request(
+    args: &CliArgs,
+    method: reqwest::Method,
+    path: &str,
+    bearer_token: Option<&str>,
+    body: Option<HashMap<&str, Value>>,
+) -> Response {
+    // add user agent of some sort.
+    let client = reqwest::blocking::Client::builder()
+        .user_agent("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36")
```

Comment:
Do we actually need this? Does the backend reject requests from specific user agents?

Reply:
We store the user agent with the session so the client can generate & display labels in the session list

Reply:
Ok, so we don't really need to fake a browser here then? It would be fine to just get whatever the Rust library sends by default?